NAS2D/Mixer/Mixer.h

```diff
@@ -41,121 +39,46 @@ namespace NAS2D
 		virtual ~Mixer() = default;
 
 	public:
-		/**
-		 * Plays a sound on the first available sound channel.
-		 *
-		 * \param	sound	A reference to a Sound Resource.
-		 */
 		virtual void playSound(const Sound& sound) = 0;
-
-		/**
-		 * Stops playing all sounds on all channels.
-		 */
 		virtual void stopSound() = 0;
-
-		/**
-		 * Pauses sound on all channels.
-		 */
 		virtual void pauseSound() = 0;
-
-		/**
-		 * Resumes sound on all channels.
-		 */
 		virtual void resumeSound() = 0;
-
-		/**
-		 * Starts playing a Music track.
-		 *
-		 * \param music	Reference to a Music Resource.
-		 * \param loops	Number of times to repeat the music.
-		 */
 		void playMusic(const Music& music, int loops = Mixer::CONTINUOUS);
```

Comment:
Along with removing the documentation, would it also make sense to remove the `loops` parameter (in a separate PR)? I feel like as long as we keep that parameter, it should perhaps have some documentation, particularly since the default is to loop continuously.

Reply:
This is one part of the API that didn't make sense to me. Shouldn't the default be to *not* loop?

Reply:
I'm thinking playlist control should be handled separate from this class. In which case, it would make sense to not loop.

I think the current design maybe assumes a single background music track as default. Maybe due to limited scope for the game assets early on.

Reply:
It also may be a function of the underlying audio engine. [An XAudio2 Buffer type](https://docs.microsoft.com/en-us/windows/win32/api/xaudio2/ns-xaudio2-xaudio2_buffer) requires setting up initial values for looping sample ranges and loop counts as well, but as mentioned,  it defaults to not looping, and playing the whole buffer.

Reply:
As per my original comment, it would probably be wise to keep the documentation until the `loops` parameter is removed.